Change all to Mixer Settler

Rick8484 opened the pull request:
Replacing centrifuge recipes with mixer settler.

Rick8484 committed:
Change all to Mixer Settler 

Be chain

Rick8484 committed:
Update ZirconiumChain.groovy

Rick8484 committed:
Update HafniumChain.groovy

Rick8484 committed:
Update NiobiumChain.groovy

Rick8484 committed:
Update TantalumChain.groovy

Rick8484 committed:
Update VanadiumChain.groovy

I am not sure about this but I think neither centrifuge nor mixer settler required for these

Rick8484 committed:
Make the recipe LV

Assumingly this was not intentional so changing it to LV

MCTian-mi commented:
\> yay 3129371 less motors to craft for centrifuges
\> wait what

Rick8484 committed:
I am sorry but am I fucking stupid ??!!

Rick8484 committed:
.

Rick8484 committed:
These are to prevent that weird flashings.

Rick8484 committed:
To prevent annoying

Rick8484 committed:
Doubt it requires centrifuge

Rick8484 committed:
Update TungstenChain.groovy

Rick8484 committed:
.

Rick8484 committed:
More changes

Rick8484 committed:
Update IodineChain.groovy

For recycle

planetme committed:
rebalance

Rick8484 committed:
Big change

planetme committed:
reduce # of cells for re-extractions

Rick8484 committed:
.

Rick8484 committed:
womp womp

Rick8484 committed:
Update GregTech.groovy

Rick8484 committed:
sowwy

bruberu commented on groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/separations/BastnasiteProcessing.groovy:
```diff
@@ -130,49 +129,54 @@ BR.recipeBuilder()
     .fluidOutputs(fluid('spent_p_five_zero_seven_extraction_mixture') * 5000)
     .duration(80)
     .EUt(120)
+    .requiredCells(2)
```
Note: I will have to update SuSyCore to allow the mixer settler to use 2 cells; it currently requires a minimum of 4.

Rick8484 committed:
Update ChemistryOverhaul.groovy

Reaction is unbalanced + there is already another recipe for it and it is the industrial methode. This is not how you make butanediol. (acc wiki)

Rick8484 committed:
delete weird ahh FeW recipes

Rick8484 committed:
I think this is how I fix the conflict ?

Rick8484 committed:
minor fixes

bruberu committed:
fix: erroring recipe
